backpack/extensions/firstorder/batch_l2_grad/conv1d.py

```diff
@@ -1,19 +1,6 @@
-from torch import einsum
+from backpack.extensions.firstorder.batch_l2_grad.convnd import BatchL2Convnd
```

Comment:
Please improve consistency with the [`core`](https://github.com/f-dangel/backpack/blob/master/backpack/core/derivatives/convnd.py#L13) by renaming `BatchL2Convnd` into `BatchL2ConvNd`